catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/TestPreStoragePlugin.java

```diff
@@ -48,6 +49,8 @@
 
     private static final String GEO_NITF = "/i_3001a.ntf";
 
+    private static final String FILESYSTEM_INVALID_CHARS = "\\/:*?<>|";
```

Comment:
Originally was focused on Windows specific invalid characters. But this should definitely be expanded for additional invalid characters. Added `&` and `;` for a more robust unit test. Assuming a period character is actually safe for file names, but expect them to be stripped out.